data/json/items/ranged/slings.json

```
@@ -122,4 +122,31 @@
     "reload": 50,
     "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "pebble": 1 }, "open_container": true } ]
   }
```

```suggestion
  },
```
Also just missing a comma here